### ISSUE TITLE
Does this work post 13.04?

### ISSUE THREAD

**Issue by mittenchops** [webbrandon/Surface-Boot-Themes#1]
I saw your post here:
http://askubuntu.com/questions/265644/dual-boot-surface-pro-with-ubuntu

With the note:
"Help me solve the themed boot-loader issue.

Before the 13.04 release you were able to custom theme the boot-loader as the below image demonstrates. The previous instructions I had for this can be found here: my github."

Is this to say that after the 13.04 these themes no longer work?  I'm considering buying a surface pro 2, and your art looks great, so I'd like to use this if it still works.

**Comment by 2bigO** [comment]
I'm also here from the same post @mittenchops mentioned. I've got a Surface Pro 2 which I like to have this on.

A few questions @webbrandon 
- Is this touch enabled?
- This will work on 12.04, right?
- Will it still work if I upgrade my kernel to 3.12.3?
- I still have the Wifi issue and was not able to solve it by either yours or other's solutions. I've posted a new thread [here](http://askubuntu.com/questions/386722/marvell-wifi-issue-on-surface-pro-2-ubuntu-12-04). Since you mentioned in the post that upgrading kernel to 3.11.0 will fix the Wifi issue, I'm assuming that 3.12.3 will also fix it. But will upgrading to 3.12.3 break this custom grub theme?

I'd love to post these on the original question page, but I don't have enough reps to comment :(

**Comment by wolfy1339** [comment]
@mittenchops @keyvan-m-sadeghi I have tested it on 14.04 LTS (Alpha). It should work fine past 13.04.

**Comment by webbrandon** [comment]
@keyvan-m-sadeghi @mittenchops So sorry I never got to your comments till now.  @wolfy1339 is correct about workable versions.
- wifi issue with surface one are now resolved.  Havn't heard about new versions of the surface having issues.
- As for older themes, depends on the theme but many things are the same.
- As for touch, I began researching this so I could add the feature but never got much further than the preliminary investigation.  
  - http://askubuntu.com/questions/267721/adding-touch-to-grub2

I know you probably don't need assistance anymore but there you go.
